### DISCUSSION TITLE
[Vote ended on 2022-08-03] Remove `google.cloud`from Ansible 8

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#121] [Votes done]
`google.cloud` looks unmaintained (#105).

Please vote with a +1 to remove it from Ansible 8 or -1 to keep it in this discussion.

**Comment by mariolenz** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by jamescassell** [comment]
+0 I've asked around if there's any resources that might be interested/available to maintain it, but expect the answer will be "no", in which case I'll change to affirmative vote.

**Comment by ssbarnea** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by markuman** [comment]
+/- 0

**Comment by briantist** [comment]
+1

**Comment by acozine** [comment]
+1, hoping we can find maintainers and reinstate it soon, but let's remove it for now.

**Comment by cidrblock** [comment]
+1 unfortunate but understood

**Comment by samccann** [comment]
8 Steering Committee votes in favor, 1 abstain. Moving this to being implemented phase. Thanks everyone!

**Reply by felixfontein** [reply]
Already counted in the issue: https://github.com/ansible-community/community-topics/issues/105#issuecomment-1203723822
